test/CollectionTest.cc

```diff
@@ -259,7 +254,7 @@ TEST_CASE_METHOD(CollectionTest, "Delete Default Collection", "[Collection]") {
     
     // Delete the default collection:
     REQUIRE(CBLDatabase_DeleteCollection(db, kCBLDefaultCollectionName, kCBLDefaultScopeName, &error));
-    CHECK(!CBLDatabase_DefaultCollection(db, &error));
+    CHECK(!defaultCollection);
```

Comment:
This is not correct because defaultCollection will still hold the original pointer. The original code is correct.

Reply:
I did miss that, will change where needed